### DISCUSSION TITLE
Can we help support this project?

### DISCUSSION THREAD

**Discussion by mklarmann** [handsontable/hyperformula#1442] [General]
I am sorry to ask, but it seems like for some time the branches are broken (do not build).

**Comment by mklarmann** [comment]
This is from your last blog post: https://stackblitz.com/edit/hf-mortgage-calculator-nj73tx?file=src%2Findex.js this might be related: https://github.com/handsontable/hyperformula/issues/1441

**Comment by sequba** [comment]
Hi @mklarmann, thank you for reaching out to us.

We are really sorry, but some of the recent issues might have been hanging longer than usual. The HyperFormula team was significantly reduced during August due to the holiday season, but the **project is still actively maintained.**

> the branches are broken (do not build)

Did you experience any issues building HyperFormula yourself?

We are aware that some of the GitHub workflows are failing due to #1430 and https://github.com/handsontable/hyperformula/issues/1428. We are working to fix these issues, but despite them, to the best of our team's knowledge, **the code in the `master` branch is stable and ready to use. As well as the last npm release.** 

> This is from your last blog post: https://stackblitz.com/edit/hf-mortgage-calculator-nj73tx?file=src%2Findex.js

What exactly is wrong with this demo? We are aware of the Stackblitz limitations that cause some issues with our demos in browsers other than Chrome and because of that we decided to do #1441

> Can we help support this project?

The HyperFormula team is always open for contributions. If you happen to be the `karma` expert, you may help us with #1430 or just take a look at the [list of "good first issues"](https://github.com/handsontable/hyperformula/labels/Good%20First%20Issue).
